tests/integration/targets/setup_sqlserver/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install Powershell modules
+  ansible.builtin.command: >
+    pwsh -Command "
+      if (-not(Get-Module -FullyQualifiedName @{ModuleName='dbatools';ModuleVersion='{{ dbatools_min_version }}'} -ListAvailable)) {
+        Install-Module dbatools -MinimumVersion {{ dbatools_min_version }} -Force
+      }"
+  no_log: true
```

Comment:
why `no_log` here?

Reply:
The install gets to be verbose (progress bar) when slow, so trying to cut down on the output.